packages/roosterjs-editor-dom/lib/selection/NormalSelectionRange.ts

```diff
@@ -0,0 +1,18 @@
+import { INormalSelectionRange, SelectionRangeTypes } from 'roosterjs-editor-types';
+
+/**
+ * Normal selection Range used in the getSelectedRangeEx editor Api
+ */
+export default class NormalSelectionRange implements INormalSelectionRange {
+    constructor(ranges: Range[]) {
+        this.ranges = ranges;
+        this.type = SelectionRangeTypes.Normal;
+    }
+
+    type: SelectionRangeTypes.Normal;
+    ranges: Range[];
+
+    isCollapsed = () => {
+        return this.ranges[0].collapsed;
```

Comment:
If we make isCollapse as a simple boolean value, no need to have this class, but just make it a simple object

Reply:
I agree, This is a better approach setting this property as a simple boolean.
Or even removing it as we could just check for each of the ranges inside of the array if the actual Range is collapsed.